setupext.py

```diff
@@ -2,13 +2,12 @@
 import configparser
 from distutils import sysconfig, version
 from distutils.core import Extension
-import distutils.command.build_ext
 import glob
 import hashlib
 import importlib
 import multiprocessing
 import os
-import pathlib
+from pathlib import Path
```

Comment:
That doesn't work: there's a `class Path` later... (even if it does that'd be quite confusing)

Reply:
The one place, where I thought it was ok to import a class directly because it's in the standard library...

Reply:
:)